src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.html

```diff
@@ -396,3 +396,60 @@
                               [text]="value">
   </cd-copy-2-clipboard-button>
 </ng-template>
+
+<ng-template #editingTpl
+             let-value="data.value"
+             let-row="data.row"
+             let-column="data.column">
+  @if (isCellEditing(row?.id, column?.prop)) {
+  <form [formGroup]="column?.customTemplateConfig?.formGroup"
+        #formDir="ngForm">
+    <div cdsRow>
+      <div cdsCol>
+        <cds-text-label [invalid]="column?.customTemplateConfig?.formGroup.controls[column?.prop].invalid && column?.customTemplateConfig?.formGroup.controls[column?.prop].dirty"
```

Comment:
```suggestion
        <cds-text-label [invalid]="showError(column?.customTemplateConfig?.formGroup.controls[column?.prop])"
```

Reply:
actually you shouldn't use function calls in angular template. 
https://medium.com/showpad-engineering/why-you-should-never-use-function-calls-in-angular-template-expressions-e1a50f9c0496